packages/blockchain-wallet-v4-frontend/src/components/Chart/hooks/useLast/useLast.ts

```diff
@@ -0,0 +1,4 @@
+import { useMemo } from 'react'
+
+export const useLast = <T extends unknown = unknown>(value: T[]): T =>
```

Comment:
can this name be more explicit?

Reply:
I will remove this hook because it doesnt make sense to have one for that